orchagent/intfsorch.cpp

```diff
@@ -378,7 +501,7 @@ void IntfsOrch::doTask(Consumer &consumer)
                     it++;
                     continue;
                 }
-                if (!vnet_orch->setIntf(alias, vnet_name, ip_prefix_in_key ? &ip_prefix : nullptr))
+                if (!vnet_orch->setIntf(alias, vnet_name, ip_prefix_in_key ? &ip_prefix : nullptr, adminUp, mtu))
```

Comment:
I'm not sure if you want to add this `adminUp `and `mtu ` across all the function signature/usage. Can we handle this differently?

Reply:
Do you have a suggestion? I did not go through the vrf case. This is what I think to be safe without knowing any control flows regarding the VRF portion of IntfOrch.